jobs/build/ocp3/Jenkinsfile

```diff
@@ -861,6 +861,7 @@ images:rebase --version v${NEW_VERSION}
 --working-dir ${DOOZER_WORKING} --group openshift-${params.BUILD_VERSION}
 ${ODCS_FLAG}
 ${exclude}
+--filter-by-os amd64
```

Comment:
it turns out `oc image mirror` effectively does this by default. it's fine to make it explicit, but see below...